qa/suites/rbd/persistent-writeback-cache/pool/rwl.yaml

```diff
@@ -0,0 +1,10 @@
+tasks:
+- install:
+- ceph:
+    conf:
+      client:
+        rbd_persistent_cache_mode: rwl
```

Comment:
Nit: should probably be in a different directory from `pool` (i.e. `cache_mode`) and we would want a file to also exercise the SSD mode.

Reply:
I created a new folder for rwl. And the other file for SSD will be created in a later PR as it needs to test at first.